packages/tocco-ui/src/EditableValue/typeEditors/SingleSelect.js

```diff
@@ -48,14 +48,9 @@ const SingleSelect = props => {
 
 SingleSelect.propTypes = {
   onChange: PropTypes.func.isRequired,
-  value: PropTypes.oneOfType([
-    PropTypes.shape({
-      key: PropTypes.oneOfType([
-        PropTypes.string,
-        PropTypes.number
-      ])
-    })
-  ]),
+  value: PropTypes.shape({
```

Comment:
same as above

  value: 
    PropTypes.shape({
      key: PropTypes.oneOfType([
        PropTypes.string,
        PropTypes.number
      ])
    }),

would be correct